src/home.js

```diff
@@ -0,0 +1,18 @@
+export default function home() {
```

Comment:
- [ ] [REQUIRED] It should use ES6 syntax (arrow functions instead of the `function` keyword in this case), **this is only an example** there are more cases in the code where you could implement ES6 syntax.
![image](https://user-images.githubusercontent.com/73254040/101965961-048b1600-3bdc-11eb-8c60-249889e89407.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)